website/addons/github/static/githubFangornConfig.js

```diff
@@ -176,7 +176,11 @@ var _githubItemButtons = {
                         },
                         icon: 'fa fa-external-link',
                         className: 'text-info'
-                    }, 'Open'),
+                    }, 'Open')
+                );
+            }
+            if (item.data.addonFullname) {
```

Comment:
This one should not contain the condition `tb.options.placement !== 'fileview'` because we always want to show the dropdown for the github branches (not just on the file view page)